widgetssdk/src/main/java/com/glia/widgets/messagecenter/MessageCenterView.kt

```diff
@@ -79,6 +81,22 @@ class MessageCenterView(
         initCallbacks()
     }
 
+    private fun setupViewAppearance() {
+        val callback: RequestCallback<Boolean> = RequestCallback { isAvailable, exception ->
+            if (exception != null) {
+                showUnexpectedErrorDialog()
+                sendMessageGroup.visibility = INVISIBLE
+                return@RequestCallback
+            }
+            if (!isAvailable) {
+                showMessageCenterUnavailableDialog()
+                sendMessageGroup.visibility = INVISIBLE
+            }
+        }
+
+        controller?.isMessageCenterAvailable(callback)
```

Comment:
I guess I will refactor it in next PR.
I prefer to move this functionality by using the State class.

Reply:
Okay, thank you!